content/influxdb/v2.0/reference/cli/influx/task/retry-failed.md

```diff
@@ -0,0 +1,92 @@
+---
+title: influx task retry-failed
+description: The `influx task retry-failed` command retries failed InfluxDB a task runs.
+menu:
+  influxdb_2_0_ref:
+    name: influx task retry-failed
+    parent: influx task
+weight: 201
+---
+
+The `influx task retry-failed` command retries failed InfluxDB a task runs.
+
```

Comment:
```suggestion
The `influx task retry-failed` command retries failed InfluxDB task runs.
```

Reply:
How about retries failed InfluxDB tasks? Curious whether adding the "run" reflects a distinction